app/Http/Controllers/ArticleController.php

```diff
@@ -58,7 +58,7 @@ public function show($id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function edit($id)
+    public function edit(int $id)
```

Comment:
not only typed params (which is also something to fix) , but return types too. So in this case would be 

use  \Illuminate\Http\Response;
....
public function edit(int $id) : Response {
}

Reply:
Can I fix this in a separate pull request after merging all the branches with previous fixes so that there are no extra files?

Reply:
absoilutely, just make sure it's fixed everywhere, not only in ArticleController